pyopenetl/operations.py

```diff
@@ -304,7 +304,38 @@ def delete_table(self, table: str) -> None:
         with self.dest_conn.connect() as cloud_sql_conn:
             print(f"--> Deleting table {table}")
             cloud_sql_conn.execute(f"DROP TABLE IF EXISTS {table}")
+    def get_indices_from_heroku(self, read_table: str, write_table: str) -> str:
+        """
+        Reads the index commands from the source heroku table, and returns a command to run on the destination table.
+
+        args:
+            read_table: the name of the heroku table being read from
+            write_table: the name of the Cloud SQL table we write to
+        """
 
+        # Query to get indices from read_table
+        data_to_resolve_query = f"""SELECT
+                                tablename,
+                                indexname,
+                                indexdef
+                            FROM
+                                pg_indexes
+                            WHERE
+                                schemaname = 'public' AND
+                                tablename = '{read_table}'
+                            ORDER BY
+                                tablename,
+                                indexname;"""
+        with self.source_conn.connect() as conn:
+            index_df = pd.read_sql(data_to_resolve_query, conn)
+
+        final_query_str = ""
+        # Clean the index queries and concatenate them into one nice (long) query
+        for query in index_df['indexdef']:
+            tempstr = query.replace(f"ON public.{read_table}", f"ON {write_table}")
```

Comment:
one thing we should consider here is what happens if these index creation queries are applied to tables in a schema other than `public` -- how can we update this to handle that? e.g., what if a query says `...ON v0r0.my_table` instead of `...ON public.my_table`?